index.js

```diff
@@ -1,13 +1,34 @@
+//Completed by Mike, Reid, Bryam, Sam, and Chloe
+
+
+
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
+header.style.color = "red"
 
 
 /***** Deliverable 3 *****/
-// console.log("PLAYERS array looks like this:", PLAYERS)
+console.log("PLAYERS array looks like this:", PLAYERS)
+let playerContainer = document.querySelector('.player-container')
+
+
+PLAYERS.forEach(function(player){
+    let playerDiv = document.createElement('div')
+    playerDiv.className = "player"
+    playerDiv.dataset.number = `${player.number}`
```

Comment:
On this line, no need for the string interpolation - since `player.number` is already a string, you can just write:

```js
playerDiv.dataset.number = player.number
```